Cargo.toml

```diff
@@ -50,7 +50,7 @@ members = [
 [workspace.dependencies]
 async-compression = "0.3.5"
 async-process = "1.0.1"
-async-std = "1.12.0"
+async-std = { version = "1.12.0", features=["tokio1"] }
```

Comment:
wait, I don't think this is the right place to do this. You should add this specific features array to Nassun's `Cargo.toml` (I'm surprised this worked at all. I could've sworn last I tried doing this, cargo yelled at me that this table was _only_ for versions)

Reply:
Maybe I look scary and Cargo didn't dare yell at me? Fixed.